FSW/src/util/timing.cpp

```diff
@@ -195,8 +211,8 @@ int wrapBufferIdx(int idx){
     Serial.println(" tries.");
     Serial.println("Defaulting to an index of 0 (Timing module)");
     
-    // TODO: Should this return -1 instead to throw an error?
-    return 0;
+    // Return -1 to throw an error since something has gone very wrong if we get here

```

Comment:
So it seems like -1 is being used as the error code in a lot of things. If we see this, will it be obvious where the code fails? Or should these -1 returns be preceded by something like `Serial.println("ouch this hurts")`

Reply:
this one is preceded by a warning "more than 10 tries, defaulting to buffer idx of 0", so yes I agree